backend/curriculum_tracking/card_generation_helpers.py

```diff
@@ -266,7 +266,14 @@ def update_cards_accorrding_to_progress(user):
 
 
 def generate_all_content_cards_for_team(team):
-    todo
+    # todo
```

Comment:
This function was meant to fail loudly instead of fail silently.
I don't think it's being used anywhere though.

If you don't like the `todo` you can explicitly raise a notimplemented exception. It amounts to the same thing. `todo` is just faster to type.